highcharts-api/highcharts/11-3d-funnels/README.md

```diff
@@ -0,0 +1,6 @@
+# Exercise 11 - 3D Funnels
+
+Create a 3D chart composition, like in the image below, using two series:
+`funnel3d` and `pyramid3d`.
```

Comment:
It sounds like the student was supposed to create some module, since there is even a module named `3dFunnelComposition` etc. I'd rephrase that, e.g. 
```Create a combination of 3D series types in the chart. ```